app/helpers/proposal_fields_helper.rb

```diff
@@ -73,4 +73,12 @@ def dates_answer(field, proposal, attr)
   def action
     params[:action] == 'show' || (params[:action] == 'location_based_fields' && request.referer.exclude?('edit'))
   end
+
+  def mendatory_field?(field)
+    field.validations.where(validation_type: 'mandatory').present?
+  end
+
+  def print_validation
+    '<span style="color:red; margin-left: 3px;">*</sapn>'.html_safe
```

Comment:
We should add a class to app/assets/stylesheets/proposal_forms.css instead of inline styles. Also, the closing tag is misspelled.